lms/models/h_user.py

```diff
@@ -33,6 +26,5 @@ class HUser(NamedTuple):
     provider_unique_id: str = ""
     """The "provider_unique_id" string to pass to the h API for this user."""
 
-    @property
-    def userid(self):
-        return f"acct:{self.username}@{self.authority}"
+    def userid(self, authority):
+        return f"acct:{self.username}@{authority}"
```

Comment:
We'll follow exactly the same technique with the new `models.HGroup` in a future PR: it'll have a `groupid(authority)` method

Reply:
Makes sense to me.